### PR TITLE
Added test suite

### PR DESCRIPTION
**Pull request by mssola**
Signed-off-by: Miquel Sabaté Solà <msabate@suse.com>

### CODE REVIEW

**Comment by mssola** [comment]
Note that I've also removed the password recovery thingie. This is because it's half-implemented anyways, and I really wanted to have 100% test coverage from now on.

File: Gemfile [hunk 1]
```diff
@@ -65,5 +65,7 @@ unless ENV["PACKAGING"] && ENV["PACKAGING"] == "yes"
     gem "codeclimate-test-reporter", "~> 1.0.0", require: nil
     gem "factory_girl_rails"
     gem "ffaker"
+    gem "rails-controller-testing"
```
Comment (jimmykarily) [inline Gemfile]: If we switch to Rails 4 we might not need this yet.

**Comment by ereslibre** [comment]
Really big cleanup and important additions! Thank you so much for this.

LGTM :+1:

File: Gemfile (hunk 1 above)
Comment (mssola) [inline Gemfile]: True. Let's do that on another review though.

File: spec/controllers/updates_controller_spec.rb [hunk 2]
```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+require "rails_helper"
+
+RSpec.describe UpdatesController, type: :controller do
+  let(:user)   { create(:user) }
```
Comment (jimmykarily) [inline spec/controllers/updates_controller_spec.rb]: [minor] too many spaces between ) and {

Comment (mssola) [inline spec/controllers/updates_controller_spec.rb]: Fixed

**Pull request merged**